packages/design-system/src/components/N8nInput/Input.vue

```diff
@@ -85,11 +85,14 @@ export default defineComponent({
 			return this.size;
 		},
 		classes(): string[] {
+			const classes = [];
 			if (this.size === 'xlarge') {
-				return ['xlarge'];
+				classes.push('xlarge');
 			}
-
-			return [];
+			if (this.type === 'password') {
```

Comment:
not every input in credentials modal is a password input.. should we handle those seperately.. for example sometimes api keys are plain text inputs..

Reply:
True, but we need to hide password inputs no matter where they are and it's done here. All other inputs in credentials modals are hidden by [this](https://github.com/n8n-io/n8n/blob/2e94e4ff652a29930bf8835ad31954e2cffaf00d/packages/editor-ui/src/components/ParameterInput.vue#L833).
Example:
<img width="342" alt="image" src="https://github.com/n8n-io/n8n/assets/2598782/935f7344-66fc-4948-a4cd-2ad9d08916e9">

Reply:
perfect